raui-core/src/widget/mod.rs

```diff
@@ -570,18 +633,16 @@ macro_rules! widget {
             $crate::widget::node::WidgetNode::Component(component)
         }
     };
-}
-
-#[macro_export]
-macro_rules! widget_wrap {
-    ({$expr:expr}) => {
+    (@wrap {$expr:expr}) => {
         $crate::widget::node::WidgetNode::from($expr)
     };
-    ($tree:tt) => {
+    (@wrap $tree:tt) => {
         $crate::widget!($tree)
```

Comment:
I removed the `widget_wrap` macro here in favor of adding internal rules to the existing `widget` macro. We could alternatively just add the `#[doc(hidden)]` flag to the `wiget_wrap` macro. I initially removed it instead of doing `doc(hidden)` because I though that it would avoid needing to have `widget_wrap` in scope, but the `$crate::widget_wrap` should fix that so it wouldn't actually be an issue.

Reply:
i have no problem with removing `widget_wrap!` since it was made purely for RAUI internals, and this way is even better <3